Security.Platform.Android/Checks/DeveloperModeDisabledCheck.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using Android.Content;
+using Android.Provider;
+
+namespace AeroGear.Mobile.Security.Checks
+{
+    /// <summary>
+    /// Security check that detects if developer mode is enabled in the device.
+    /// </summary>
+    public class DeveloperModeDisabledCheck : AbstractSecurityCheck
+    {
+        protected override string Name => "Developer Mode Check";
+
+        private readonly Context context;
+
+        public DeveloperModeDisabledCheck(Context ctx)
+        {
+            this.context = ctx;
+        }
+
+        public override SecurityCheckResult Check()
+        {
+            bool devModeEnabled = Settings.Secure.GetInt(
```

Comment:
When running this in an emulator I'm seeing a `SettingNotFoundException` being thrown.

```
[AndroidRuntime] android.runtime.JavaProxyThrowable: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> Android.Provider.Settings+SettingNotFoundException: development_settings_enabled
[AndroidRuntime]   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <f32579baafc1404fa37ba3ec1abdc0bd>:0 
[AndroidRuntime]   at Java.Interop.JniEnvironment+StaticMethods.CallStaticIntMethod (Java.Interop.JniObjectReference type, Java.Interop.JniMethodInfo method, Java.Interop.JniArgumentValue* args) [0x00069] in <09bf3e262b934ffab2ba01f9fc7fd54d>:0 
[AndroidRuntime]   at Java.Interop.JniPeerMembers+JniStaticMethods.InvokeInt32Method (System.String encodedMember, Java.Interop.JniArgumentValue* parameters) [0x00018] in <09bf3e262b934ffab2ba01f9fc7fd54d>:0 
[AndroidRuntime]   at Android.Provider.Settings+Secure.GetInt (Android.Content.ContentResolver cr, System.String name) [0x0004b] in <25661073a35344a89f215a4cf81af37c>:0 
[AndroidRuntime]   at AeroGear.Mobile.Security.Checks.DeveloperModeDisabledCheck.Check () [0x0000c] in <3af04addddc1407fafe20c72004681b5>:0 
[AndroidRuntime]   at AeroGear.Mobile.Security.Executors.Sync.SyncSecurityCheckExecutor.Execute () [0x0001f] in <0e8c095b6cac4364a4d9747999250031>:0 
```

Reply:
Should be fixed now.